pkg/operator/apiserver/audit/audit_policies_test.go

```diff
@@ -155,6 +155,29 @@ func TestNewAuditPolicyPathGetter(t *testing.T) {
 	}
 }
 
+func TestDafaultPolicy(t *testing.T) {
+	scenarios := []struct {
+		name            string
+		targetNamespace string
+		targetName      string
+		goldenFile      string
+		expectedPath    string
+	}{
+		{
+			name: "Get default audit policy",
+			targetNamespace: "ScenarioOne",
+			targetName:      "audit",
+			goldenFile:      "./testdata/audit-policies-cm-scenario-1.yaml",
+			expectedPath: "/var/run/configmaps/audit/default.yaml"
+		},
+	}
+	{
+		// act
+		cm, err := GetAuditPolicies(scenarios.targetName, scenarios.targetNamespace)
```

Comment:
we are adding a new function `DefaultPolicy` which should be tested I believe, not `GetAuditPolicies`.